core/src/main/java/io/fabric8/maven/core/service/openshift/OpenshiftBuildService.java

```diff
@@ -118,6 +110,11 @@ public void build(ImageConfiguration imageConfig) throws Fabric8ServiceException
         } catch (Fabric8ServiceException e) {
             throw e;
         } catch (Exception ex) {
+            // Log additional details in case of any IOException
+            if (ex != null && ex.getCause() instanceof IOException && ex.getCause().getMessage().contains("Socket closed")) {
```

Comment:
It would be better, either get rid of `.contains("socket closed")` or do string case insensitive comparison `ex.getCause().getMessage().toLowerCase().contains("socket closed")`

Reply:
@rohanKanojia any update on this?